vsts/pipelines/templates/_signBinary.yml

```diff
@@ -52,6 +52,14 @@ steps:
     configuration: '$(BuildConfiguration)'
     condition: and(succeeded(), eq(variables['setSignTypeVariable.SignType'], 'real'))
 
+- task: VSBuild@1
+  displayName: 'Sign Build Server Binaries'
+  inputs:
+    solution: 'src/BuildServer/BuildServer.signproj'
+    msbuildArgs: '/t:SignFiles /p:RuntimeIdentifier=linux-x64 /p:MicroBuild_SigningEnabled=true'
+    configuration: '$(BuildConfiguration)'
+    condition: and(succeeded(), eq(variables['setSignTypeVariable.SignType'], 'real'))
+
```

Comment:
If you look at lines 63-82, those two tasks also process the binaries.

Reply:
We need to add those for BuildServer as well. After that I'm wondering if we should trigger CI build for this branch and test it? Since this _signBinary.yml file only appears in CI build, there's no way to test other than having a CI build.

Reply:
That's true Oryx-CI is the only pipeline that does a release